### DISCUSSION TITLE
About use LiveCharts2 in production

### DISCUSSION THREAD

**Discussion by jldearmas** [Live-Charts/LiveCharts2#1184] [Announcements]
I'm looking for a package to plot biomedical signals in real-time, and I'm evaluating this library, but I would like more about use it in a production project.

- Is the last beta of version 2.0 recommended for use in production?
- Is there any idea of a release date for the 2.0 final version?

Thanks in advance

JL

**Comment by Kevin-Seiden** [comment]
It can be used for production, however as it is in Beta, you have to remember that things can change wildly and what you code now may have to be significantly re-coded to update to the next version.  However, there is no requirement for you to immediately upgrade, NuGet has the historical packages.  I still have a project on beta 104 because I need to rewrite the tooltip template that changed.  But in the end it's up to you, nobody can make the determination of whether you are willing to do that but you.

**Comment by beto-rodriguez** [comment]
Currently most of the issues seem fixed, I don't expect big changes in the way to 2.0, about the date, I hope on November of this year we have the first GA version.

**Reply by jldearmas** [reply]
Sounds good... I'll start making some tests. I'm planning to use library in applications for two platforms:

- Windows

- Android

**Comment by jeetee** [comment]
We're also looking into possibly using LiveCharts2 in our company, but would need to handle around 2M data points, which simply isn't viable in the current state of the project.
I'm trying to figure out whether we'd have to look into other libraries / invest time into developing our own virtualization / await the release of v2 (for which [high performance](https://github.com/beto-rodriguez/LiveCharts2/issues/486) looks promising for our needs).

@beto-rodriguez Can you share any insight into an updated release expectancy schedule?

**Comment by beto-rodriguez** [comment]
The Expected release date is in February/March ff this year.

**Reply by jeetee** [reply]
We're aiming for a first product release around March as well; any chance we could get beta/preview access to the high performance build?

**Reply by gvinson1** [reply]
Any update on the release date or schedule?

**Reply by jeetee** [reply]
Due to lack of maintenance on this project, we've decided as a company to move on to a graphing library that seems to be somewhat better maintained: https://github.com/ScottPlot/ScottPlot

LiveCharts2 looked very promising; perhaps when it finally comes to fruition we can consider it for a future project.

**Reply by RobSmyth** [reply]
Has there been any update on a 2.0.0 release? I see it has been in beta/RC for a few years and I also see @jeetee 's comment Jul 8, 2024.

If no expectation of a 2.0.0 release, could you give an idea of what is blocking "RC" it from release. I'm hoping this will help understanding if fit for production. 

Thanks
